pkg/machines/getOSList.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python3
+
+import gi
+gi.require_version('Libosinfo', '1.0')
+from gi.repository import Libosinfo
+import sys
+
+
+class _OsinfoIter:
```

Comment:
Too much overhead IMHO, the simple iteration was fine. I just wanted to know why it was done that way.